build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     jcenter()
   }
   dependencies {
-    classpath "com.android.tools.build:gradle:3.1.4"
+    classpath 'com.android.tools.build:gradle:3.2.1'
 
```

Comment:
In this file also update the `androidGradlePluginVersion`.

Reply:
@ritik2017 , you could squash commits and push force them so the history only shows the relevant change, one single commit changing the version in 2 places, a 2 line change. now it shows the commit of the search page change, and the revert, 6 commits in total.

Reply:
Okay doing that here